BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -347,7 +361,38 @@ open class SBAScheduledActivityManager: NSObject, SBASharedInfoController, ORKTa
             taskViewController.task?.resetTrackedDataChanges()
         }
         
-        taskViewController.dismiss(animated: true) {}
+        taskViewController.dismiss(animated: true) {
+            self.offMainQueue.async {
+                self.deleteOutputDirectory(for: taskViewController)
+                #if DEBUG
+                DispatchQueue.main.async {
```

Comment:
I've had issues in the past with logging to the console from random threads leaving some debug output interleaved with other output, making it confusing and hard to follow. Not sure it would be a problem here, or that this would even actually help with that if it were though, so... yeah. You want I should remove it?